.gitignore

```diff
@@ -15,5 +15,6 @@ out/
 .classpath
 node_modules
 .run
-run.bat
+**/run.bat
 *.sh
+/server/src/main/resources/secrets/*
```

Comment:
This shouldn't be necessary...secrets aren't a thing anymore. If you have them locally, delete them.